Sources/CXUtility/Atom.swift

```diff
@@ -2,6 +2,14 @@ public final class Atom<Val> {
     
     private let lock = Lock()
     private var val: Val
+
+    var isMutating: Bool {
+        if lock.try() {
```

Comment:
Build failed here.

Reply:
@FranzBusch Your changes to the `Lock` was lost, :-D

Reply:
Ah will fix it